python/brayns/network/instance.py

```diff
@@ -22,7 +22,7 @@
 
 from typing import Any, Protocol, TypeVar
 
-from .jsonrpc import JsonRpcFuture, JsonRpcRequest
+from .jsonrpc import JsonRpcFuture, JsonRpcReply, JsonRpcRequest
```

Comment:
Just as curiosity (and its not from this PR), Reply is used for human to human communication. Response is more correct for communication with machines.